### PR TITLE
Fix git commands completion

### PR DESCRIPTION
**Pull request by avlach**
This one fixes the git completion for main commands (e.g. git {TAB][TAB])

### CODE REVIEW

File: completions/git.completion.sh [hunk 1]
```diff
@@ -624,7 +624,7 @@ __git_commands () {
 	then
 		printf "%s" "${GIT_TESTING_COMMAND_COMPLETION}"
 	else
-		git help -a|egrep '^  [a-zA-Z0-9]'
+		git help -a|egrep "^  [a-z]*"|cut -d" " -f4
```
Comment (nntoan) [inline completions/git.completion.sh]: @avlach Will this fix POSIX-compatible?

Comment (avlach) [reply completions/git.completion.sh]: Yes, this work with bash under posix mode. I have tested with latest Fedora release 30. 

~~~
wasp  ~  bash --posix 
bash-5.0$ bash --version 
GNU bash, version 5.0.7(1)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
bash-5.0$ git help -a|egrep "^  [a-z]*"|cut -d" " -f4
add
am
archive
bisect
branch
...
stripspace
~~~

**Comment by hashlash** [comment]
Hi,

I'm sorry but that doesn't work on my machine. I don't know if it caused by different bash, git, cut, or egrep version

```
hashlash@hashlash-K401UQK:~$ bash --posix 
bash-4.3$ bash --version
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
bash-4.3$ git help -a|egrep "^  [a-z]*"|cut -d" " -f4
...
<empty lines only>
...
```

But using third field on cut works out for me
```
bash-4.3$ git help -a|egrep "^  [a-z]*"|cut -d" " -f3




add
add--interactive
am
annotate
apply
...
merge-one-file
```

Note that it only show git command from `add` to `merge-one-file`. The following commands (`merge-ours` until `write-tree`) is not showing up

```
bash-4.3$ git --version
git version 2.7.4

bash-4.3$ egrep --version
grep (GNU grep) 2.25
...
bash-4.3$ cut --version
cut (GNU coreutils) 8.25
...
```

**Comment by Kreyren** [comment]
@avlach requesting feedback on #65

**Pull request merged**